src/synth/constants.h

```diff
@@ -160,5 +160,5 @@ enum
 #endif
 
 #ifdef _WIN32
-    #define snprintf _snprintf
+    //#define snprintf _snprintf
```

Comment:
I had to comment this out as it was failing to build under MSYS2 MinGW-w64.  This seems like a dirty hack anyway.